tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1041,11 +1052,26 @@ When('the user uploads overwriting file {string} using the webUI', function(file
 When(
   'the user tries to create a file with already existing name {string} using the webUI',
   function(fileName) {
-    return client.page.personalPage().triesToCreateExistingFile(fileName)
+    return (
+      client.page
+        .personalPage()
+        .triesToCreateExistingFile(fileName)
+        // while creating resource, after clicking the "Create" button, the button should disappear but if it doesn't
+        // we throw "ElementPresentError" error. So, all the error except "ElementPresentError" is caught and thrown back
+        // Also, when no error is thrown, the button seems to disappear, so an error should be thrown in such case as well.
+        .then(() => {
+          throw new Error('ElementPresentError')
+        })
+        .catch(err => {
+          if (err.message !== 'ElementPresentError') {
+            throw new Error(err)
+          }
```

Comment:
now `.triesToCreateExistingFile` does not throw such error imo

Reply:
I think it's okay to remove `try catch ` in this section as `waitForElementPresent` in `triesToCreateExistingFile` will throw `Element not present` error in itself?

Reply:
yes, it should throw the error if the element is not present, the error type or texts may be different